plugins/kubernetes/app/views/samson_kubernetes/_deploy_tab_nav.html.erb

```diff
@@ -1,3 +1,3 @@
-<% if @deploy.kubernetes %>
+<% if @deploy&.kubernetes || @stage&.kubernetes? %>
```

Comment:
if the deploy was not done on kubernetes we should not show manifests ...

Reply:
don't understand...

Reply:
possible for deploy to be nil.

Reply:
we show the tab nav for old deploys too, so one edge-case is that an old deploy might be non-kubernetes, but the stage no longer is, then leading to issue ... but not that likely, so not important

would the deploy just be new and not nil maybe ?

Reply:
updated to 
```
<% if @deploy ? @deploy.kubernetes : @stage&.kubernetes? %>
```